Standards/scs-01xx-v1-dns.md

```diff
@@ -0,0 +1,170 @@
+---
+title: DNS Standard
+type: Standard
+status: Draft
```

Comment:
The document should begin its life in `Proposal` state.

Reply:
Given the elaborate discussion that already happened in the corresponding issue, this PR as well as various community calls and breakout sessions, I think we are well beyond `Proposal` already.

Furthermore, I think the current process[^1] is not very clear when the transition between `Proposal` and `Draft` is supposed to happen. As I read it, the PR must be in `Draft` before the very first PR/merge anyway. But that's another topic.

[^1]: https://github.com/SovereignCloudStack/standards/blob/90f3244e4776a6d5a33b6402c5550826574a32bc/Standards/scs-0001-v1-sovereign-cloud-standards.md#process